### PR TITLE
After Borys Code Review -> Delete empty line

### PR DESCRIPTION
**Pull request by Nataniel-Antosik**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -30,6 +30,5 @@
             app:layout_constraintStart_toStartOf="parent"
             app:menu="@menu/bottom_navigation_menu" />
```
Comment (marcinWisnia) [inline app/src/main/res/layout/activity_main.xml:32]: i would completely drop any empty line here, as we have closing elements here

Comment (Nataniel-Antosik) [reply app/src/main/res/layout/activity_main.xml:32]: Ok, I see it but I think this style with empty separate line make it more readable and I think it is not incorrect. I believe I had the same style in every XML file.  Do you @marcinWisnia  and @BorysChajdas think I should in every layout file delete the empty line?

Comment (BorysChajdas) [reply app/src/main/res/layout/activity_main.xml:32]: For me it's redundant but also not very important, so for me it's up to you 😄

**Review by BorysChajdas — APPROVED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (Nataniel-Antosik) [reply app/src/main/res/layout/activity_main.xml:32]: I created an issue for this #163.

**Pull request merged**